tests-integration/Cargo.toml

```diff
@@ -28,7 +28,7 @@ common-recordbatch = { workspace = true }
 common-runtime = { workspace = true }
 common-telemetry = { workspace = true }
 common-test-util = { workspace = true }
-datanode = { workspace = true }
+datanode = { workspace = true, features = ["testing"] }
```

Comment:
question: Why do we need this feature now?

Reply:
This feature enable in test-interagtion to disable the creation of prometheus metrics collector in datanode. Because test-interagtion will create multiple object stores in test, cause panic of test.